[Currency Exchange]: Reworded task explanations (#2626)
exercism/python: exercises/concept/currency-exchange/.docs/instructions.md

Create the `exchange_money()` function, taking 2 parameters:

1. `budget` : The amount of money you are planning to exchange.

> 2. `exchange_rate` : The amount of domestic currency that equals one unit of foreign currency.

Reviewer: Should this be?
```suggestion
2. `exchange_rate` : The amount of domestic currency that equals to one unit of foreign currency.
```
I think either of them are proper English, but I am not sure. What's your thought?

Reviewer 2: Either the `that` or the `to` should go (I think) grammatically.  So one of these two (_emphasis mine_).  I agree that your phrasing (_so the second option_) sounds better:

> `exchange_rate` : The amount of domestic currency _**equal to**_ one unit of foreign currency.

OR

> `exchange_rate` : The amount of domestic currency **_that equals_** one unit of foreign currency.

Reviewer 2: Third option:

> `exchange_rate` : The amount of domestic currency **_that is equal to_** one unit of foreign currency.

But that's super-formal, and feels a bit awkward to me.

Reviewer: IMO, this might sound the best. We just need a fix right now, I honestly don't think we have to be stuck on a single sentence for something we are going to rewrite anyway. 
```suggestion
2. `exchange_rate` : The amount of domestic currency equal to one unit of foreign currency.
```

This function should return the value of the exchanged currency.

> **Note:** If there are converting USD to EUR, and there are `1.20 USD` in `1.00 EUR`, the exchange rate is `1.20`.

Reviewer 2: ```suggestion
**Note:** If your currency is USD and you want to exchange USD for EUR with an exchange rate of `1.20`, then `1.20 USD == 1 EUR`.
```

1. `denomination` : The value of a single bill.
2. `number_of_bills` : Amount of bills you received.

> This exchanging booth only deals in cash. The total you receive must be divisible by the value of one bill, which can leave behind a reminder. This function should return the total value of the given *bills*.

Reviewer 2: ```suggestion
This exchanging booth only deals in cash of certain increments. 
The total you receive must be divisible by the value of one "bill" or unit, which can leave behind a fraction or remainder.
Your function should return only the total value of the bills (_excluding fractional amounts_) the booth would give back.
Unfortunately, the booth gets to keep the remainder/change as an added bonus. 
```

Create the `get_number_of_bills()` function, taking `budget` and `denomination`.

> This function should return the *number of bills* that you can get using the *budget*. In other words, how many bills of the denomination's value fit into the total budget

Reviewer 2: ```suggestion
This function should return the _number of new currency bills_ that you can receive within the given _budget_. 
In other words, how many _whole bills_ of the new currency fit into the amount of old currency you currently have?
```

> **Note: ** You can only receive _whole bills_, not fractions of bills,  so remember to divide accordingly. You are rounding down to the nearest whole number.

Reviewer 2: ```suggestion
**Note: ** You can only receive _whole bills_, not fractions of bills,  so remember to divide accordingly. 
Effectively, you are rounding _down_ to the nearest whole bill/denomination.
```

Create the `get_number_of_bills()` function, taking `budget` and `denomination`.

This function should return the _number of new currency bills_ that you can receive within the given _budget_. 

> In other words, how many _whole bills_ of the new currency fit into the amount of old currency you currently have?

Reviewer 2: ```suggestion
In other words:  How many _whole bills_ of new currency fit into the amount of old currency you have in your budget? 
```

> **Note: ** You can only receive _whole bills_, not fractions of bills,  so remember to divide accordingly. 

Reviewer 2: ```suggestion
Remember -- you can only receive _whole bills_, not fractions of bills,  so remember to divide accordingly. 
```

> Remember -- you can only receive _whole bills_, not fractions of bills,  so remember to divide accordingly. 

Reviewer: ```suggestion
Remember -- you can only receive _whole bills_, not fractions of bills, so remember to divide accordingly. 
```
💙 Someone might get annoyed by it. 💙

Create the `exchangeable_value()` function, taking `budget`, `exchange_rate`, `spread`, and `denomination`.

> Parameter `spread` is the *percentage taken* as an exchange fee, written as an integer. It needs to be converted to decimal by dividing it by 100. 

Reviewer 2: ```suggestion
Parameter `spread` is the *percentage taken* as an exchange fee, written as an integer. 
It needs to be converted to decimal by dividing it by 100. 
```

Create the `exchangeable_value()` function, taking `budget`, `exchange_rate`, `spread`, and `denomination`.

Parameter `spread` is the *percentage taken* as an exchange fee, written as an integer. 
It needs to be converted to decimal by dividing it by 100. 

> If `1.00 EUR == 1.20 USD` and the *spread* is `10`, the actual exchange rate will be: `1.00 EUR == 1.32 USD` because 10% of 1.20 is 0.12, and this additional fee is added to the exchange.

Reviewer: For the time being this is fine. _But_ we are not here to explain basic math to all students. It's important for people to know how to deal with basic `text exercises`.

As soon as I find a good topic to rewrite this exercise, I will fix 🌟 all this 🌟 .

Reviewer 2: 💜   _And you will get aaaallll the sparkle points for rewriting!!_  I agree we aren't here to explain how to break down word problems, or how to deal with general math problems.  At some point, we _may_ want to do some short pieces or vids or walk-throughs on word-problem strategies.....but later.  😄

Create the `non_exchangeable_value()` function, taking `budget`, `exchange_rate`, `spread`, and `denomination`.

> This function should return the value that is *not* exchangeable due to the *denomination* of the bills, by subtracting your value after exchange from the theoretical value. The theoretical value assumes bills can be subdivided, therefore ignoring denomination, but does take into account spread.

Reviewer 2: ```suggestion
This function should return the value that is *not* exchangeable due to the *denomination* of the bills.
Remember - this booth gets to keep the change _in addition_ to charging an exchange fee.
Start by calculating the value you would receive if you were able to keep subdivided bills, then subtract the amount you would receive in whole bills. 
Both amounts should take the spread, or the exchange fee into account.
```
